library/network/src/lib/y2firewall/firewalld/api/zones.rb

```diff
@@ -234,7 +233,9 @@ def add_protocol(zone, protocol, permanent: permanent?)
         #   modifies the permanent configuration
         # @return [Boolean] True if service was removed from zone
         def remove_service(zone, service, permanent: permanent?)
-          modify_command("--zone=#{zone}", "--remove-service=#{service}", permanent: permanent)
+          remove_arg = offline? ? "--remove-service-from-zone" : "--remove-service"
+          modify_command("--zone=#{zone}", "#{remove_arg}=#{service}",
```

Comment:
np: just for consistency, why not put the `permanent` argument in the same line?

Reply:
To be honest, the CLI interface does not look consistent to me. But I may be missing something.

Reply:
Already pushed